src/main/java/org/folio/oaipmh/client/SourceStorageSourceRecordsClient.java

```diff
@@ -69,8 +72,10 @@ public void postSourceStorageSourceRecords(String idType, Boolean deleted, List
       buffer.appendString(ClientHelpers.pojo2json(List));
     }
 
-    HttpClientRequest request = this.httpClient.postAbs(this.okapiUrl + "/source-storage/source-records" + queryParams.toString());
+    HttpClientRequest request = this.httpClient.postAbs(this.okapiUrl + GLOBAL_PATH + queryParams.toString());
```

Comment:
I realize this was already there and you're just using it but `GLOBAL_PATH` is a lousy name for this.  Something like `SOURCE_RECORDS_PATH` is IMO much more appropriate